src/index.ts

```diff
@@ -227,6 +227,13 @@ const transform = async ({
             "$1true$3"
           );
 
+          // The workers runtime does not implement certain properties like `mode` or `credentials`.
+          // Due to this, we need to replace them with null so that request deduping cache key generation will work.
+          contents = contents.replace(
+            /(?:(JSON\.stringify\(\[\w+\.method\S+,)\w+\.mode(,\S+,)\w+\.credentials(,\S+,)\w+\.integrity(\]\)))/gm,
+            "$1null$2null$3null$4"
+          );
+
```

Comment:
This is quite brittle, could you add a TODO to fix this like the `contents.replace` above?

Also I think it would be valuable if you could add a link to the code you're patching (https://github.com/vercel/next.js/blob/cbd3c2983ab8730594d7a9633f56a39c1e5b920f/packages/next/src/compiled/react/cjs/react.shared-subset.development.js#L198)